doc-www/Style-Guide.md

```diff
@@ -0,0 +1,75 @@
+ompi-www Style Guide
+===========================
+
+This is a brief Style Guide for code and prose to be included on the Open MPI web site
+(which is contained in this `ompi-www` repo).
+
+
+# Code Style
+
+## HTML and PHP Style
+
+* Manually hard-wrap lines at about 80 characters
+
+## HTML Style
+
+* Tag and attribute names in lower case
+* Otherwise, do whatever works, I guess
+
+## PHP Style
+
+* ...???... I have no idea what PHP style should look like -apjanke ...???...
```

Comment:
Ditto (I have no idea what PHP style should look like).

I guess we mainly want to avoid PHP warnings that are emitted in the web server logs...?  (e.g., using variables before they are defined, yadda yadda yadda)